.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 20.11.0
+ruby 2.7.6
```

Comment:
plugin versions list for asdf users.
like `.nvmrc`, it is directory specific, and is applied recursively to subdirectories automatically.  
https://asdf-vm.com/manage/configuration.html#tool-versions
- upside: it will of course be ignored by nvm and rbenv
- downside: it may need to be added to the checklist of files to update whenever you upgrade nodejs or ruby